tests/validator_test.go

```diff
@@ -775,6 +791,21 @@ func eventuallyFailToCreateVm(vm *kubevirtv1.VirtualMachine) bool {
 	}, shortTimeout).Should(Equal(metav1.StatusReasonInvalid), "Should have given the invalid error")
 }
 
+func expectTotalRejectedCountToIncBy(diff int) func() {
+	rejectedBefore := totalRejectedVmsMetricsValue()
+	return func() { Expect(totalRejectedVmsMetricsValue() - rejectedBefore).To(Equal(diff)) }
+}
+
+func totalRejectedVmsMetricsValue() (sum int) {
+	pods, err := GetRunningPodsByLabel(validator.VirtTemplateValidator, validator.KubevirtIo, strategy.GetNamespace())
+	Expect(err).ToNot(HaveOccurred(), "Could not find the validator pods")
+	Expect(len(pods.Items)).To(BeNumerically(">", 0))
```

Comment:
You can use `Expect(pods.Items).ToNot(BeEmpty())`.

Reply:
I can, and I will :)